client/ayon_houdini/plugins/publish/collect_local_render_instances.py

```diff
@@ -62,23 +65,23 @@ def process(self, instance):
 
         # Create Instance for each AOV.
         context = instance.context
-        expectedFiles = next(iter(instance.data["expectedFiles"]), {})
+        expected_files = next(iter(instance.data["expectedFiles"]), {})
 
         product_type = "render"  # is always render
-        product_group = get_product_name(
-            context.data["projectName"],
-            context.data["taskEntity"]["name"],
-            context.data["taskEntity"]["taskType"],
-            context.data["hostName"],
-            product_type,
-            instance.data["productName"]
-        )
-
-        for aov_name, aov_filepaths in expectedFiles.items():
-            product_name = product_group
 
+        for aov_name, aov_filepaths in expected_files.items():
+            dynamic_data = {}
             if aov_name:
-                product_name = "{}_{}".format(product_name, aov_name)
+                dynamic_data["aov"] = aov_name
```

Comment:
It misses render layer dynamic data. 

```suggestion
            dynamic_data = {
                "renderlayer": instance.data["renderlayer"]
            }
            if aov_name:
                dynamic_data["aov"] = aov_name
```

Reply:
I've added *renderlayer* in optional way so it is not broken whenever this plugin changes families (and the render layer collector doesn't). Dynamic data shouldn't be mandatory anyway.